tools/CheckDates.scala

```diff
@@ -0,0 +1,184 @@
+import sys.process._
+
+object CheckDates {
+  val usage = """
+
+    This tool compares a copyright date in files with the date of last change in git.
+    If the message of last change match regex "\\[*.\\]" (e.g. '[Tech message]')  it will be ignored and 
```

Comment:
This regex is looser than required.

```
scala> val techPat = "\\[*.\\]".r
techPat: scala.util.matching.Regex = \[*.\]

scala> techPat findFirstIn "aa [Sync copyright dates] bb"
res0: Option[String] = Some(s])

scala> techPat findFirstIn "aa dates] bb"
res1: Option[String] = Some(s])
```

Drop the over generalisation and look for a fixed string. Here is my recurring candidate: `[Sync copyright dates]`.